### DISCUSSION TITLE
Convert command

### DISCUSSION THREAD

**Discussion by Afshin1980** [explosion/spaCy#7170] [Help: Coding & Implementations] [answered]
Hello
I am quite a newbie in spaCy and I'm trying to use the command line to train a model but when I use the convert command I get the following error message I downloaded the 'ner-token-per-line.json' sample file from your repository.

python -m spacy convert ner-token-per-line.json data.spacy
Error: Invalid value for '[OUTPUT_DIR]': Path 'data.spacy' does not exist.

- **spaCy version:** 3.0.3
- **Platform:** Windows-10-10.0.18362-SP0
- **Python version:** 3.9.1
- **Pipelines:** en_core_web_lg (3.0.0), en_core_web_sm (3.0.0)

**Comment by svlandeg** [comment]
Hi! These type of general usage questions are perfectly suited for the forum, so I'll move your question there. You may get a notification that this was closed/locked, but the thread will remain open on the forum!

**Comment by svlandeg** [comment, marked as answer]
You can find the details of the `convert` command here: https://spacy.io/api/cli#convert
Specifically, the second argument (`OUTPUT_DIR`) should be a directory, not a file. Could you retry by specifying a directory name instead of `data.spacy`?

**Reply by Afshin1980** [reply]
Worked! 
Thank you
